src/arrayMethodJoin.js

```diff
@@ -5,8 +5,27 @@
  *
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function() {
-    // write code here
+  [].__proto__.join2 = function(symbol) {
+    let length = this.length;
+    // if recieved empty array , returns empty string
```

Comment:
It's clear what is happening here and this comment is not very useful. Remove the comment.

Reply:
Done.